src/app/issuer/components/badgeclass-edit-form/badgeclass-edit-form.component.html

```diff
@@ -257,9 +258,13 @@ <h2 class="tw-whitespace-pre-wrap tw-text-sm tw-text-oebblack tw-font-bold">
 			</li>
 			<hr class="tw-my-8 tw-border tw-border-[var(--color-purple)] tw-border-dashed" />
 			<li>
```

Comment:
Bullet point needs to be adjusted.
<img width="1013" alt="Screenshot 2024-07-25 at 12 31 16" src="https://github.com/user-attachments/assets/11fffbc2-c981-4aee-bda3-f776fc198f56">